lib/webserver.js

```diff
@@ -94,17 +94,17 @@ WebServer.prototype = {
     server.on('error',(e)=> {
       switch (e.code) {
       case 'EADDRINUSE':
-        networkLogger.severe(`Could not listen on address ${ipAddress}:${port}. It is already in use by another process.`);
+        networkLogger.severe(`ZWED0004E`, ipAddress, port); //networkLogger.severe(`Could not listen on address ${ipAddress}:${port}. It is already in use by another process.`);
         process.exit(constants.EXIT_HTTPS_LOAD);
         break;
       case 'ENOTFOUND':
       case 'EADDRNOTAVAIL':
-        networkLogger.severe(`Could not listen on address ${ipAddress}:${port}. Invalid IP for this system.`);
+        networkLogger.severe(`ZWED0005E`, ipAddress, port); //networkLogger.severe(`Could not listen on address ${ipAddress}:${port}. Invalid IP for this system.`);
         process.exit(constants.EXIT_HTTPS_LOAD);
         break;
       default:
-        networkLogger.warn(`Unexpected error on server ${ipAddress}:${port}. E=${e}. Stack trace follows.`);
-        networkLogger.warn(e.stack);
+        networkLogger.warn(`ZWED0071W`, ipAddress, port, e.message, e.stack); //networkLogger.warn(`Unexpected error on server ${ipAddress}:${port}. E=${e}. Stack trace follows.`);
```

Comment:
Maps to 
  "ZWED0071W":"Unexpected error on server %s:%s. E=%s. Stack trace follows.\n%s",
I don't think e.stack prints out properly with %s. It is better as:
"ZWED0071W":"Unexpected error on server %s:%s. E=%s. Stack trace follows.\n"

Reply:
Actually, I'm wrong!
It prints just fine.

Reply:
Verified and approved.